src/block-management/block-manager.js

```diff
@@ -291,13 +291,17 @@ export default compose( [
 	withDispatch( ( dispatch ) => {
 		const {
 			clearSelectedBlock,
+			editPost,
 			insertBlock,
 			replaceBlock,
 			selectBlock,
 		} = dispatch( 'core/editor' );
 
 		return {
 			insertBlock,
+			editTitle( title ) {
+				editPost( { title: title } );
+			},
```

Comment:
this might be moved to MainApp.js, what do you think? it looks like we don't call this directly from here, we just pass it to MainApp

Reply:
This is called right above in line 169.  The code in here makes `editTitle` become a prop of this object.